### PR TITLE
feat: collapsable faq sections

### PR DESCRIPTION
**Pull request by Jaszkowic**
Bildschirmfoto 2023-08-03 um 14 26 08

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204994598608180

### CODE REVIEW

File: src/components/FaqList.tsx [hunk 1]
```diff
@@ -181,7 +181,10 @@ const FAQS: {
     answer: (
       <>
         <p>
-        HILF-MIR Berlin ist ein Angebot des <a href="https://www.citylab-berlin.org">CityLAB Berlin</a> und wurde durch die Senatsverwaltung für Wissenschaft, Pflege und Gleichstellung unterstützt.
+          HILF-MIR Berlin ist ein Angebot des{' '}
```
Comment (Jaszkowic) [inline src/components/FaqList.tsx:184]: This change was done by prettier

**Comment by vogelino** [comment]
@Jaszkowic PRs should be done against `staging`. I'll change that myself and start a review!

**Comment by vogelino** [comment]
@Jaszkowic You should also merge staging into your branch before asking for a review. Just so you know. I'll do that myself this time. Cheers

**Comment by Jaszkowic** [comment]
whoops, sorry! didn't know this project had a staging environment :) thanks!

**Review by vogelino — APPROVED** [review]
Cool! Thx! I added Headless UI components to add a11y and tweaked the styles

**Pull request merged**